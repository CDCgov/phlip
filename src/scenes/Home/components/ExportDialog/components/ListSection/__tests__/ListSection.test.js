import React from 'react'
import { shallow } from 'enzyme'
import { ListSection } from '../index'

const props = {
  users: [],
  section: 'numeric',
  expanded: true,
  sectionText: 'Coded Data - Numeric',
  onExport: jest.fn(),
  onExpand: jest.fn(),
  inProgress: {
    type: null,
    user: null
  }
}

describe('Home - Export Dialog - List Section', () => {
  test('should render correctly', () => {
    expect(shallow(<ListSection {...props} />)).toMatchSnapshot()
  })
})
