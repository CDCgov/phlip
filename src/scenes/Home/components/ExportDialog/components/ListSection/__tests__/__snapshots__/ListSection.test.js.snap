// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home - Export Dialog - List Section should render correctly 1`] = `
<Fragment>
  <WithStyles(ListItem)
    selected={true}
  >
    <WithStyles(ListItemText)
      primary="Coded Data - Numeric"
    />
    <IconButton
      color="white"
      disabled={false}
      iconSize={24}
      iconStyle={
        Object {
          "color": "black",
        }
      }
      tooltipText=""
    >
      expand_less
    </IconButton>
  </WithStyles(ListItem)>
  <WithStyles(Collapse)
    in={true}
  >
    <WithStyles(List)
      component="div"
      disablePadding={true}
    >
      <ListItem
        disabled={false}
        inProgress={false}
        isSubItem={true}
        isUser={false}
        item={
          Object {
            "avatarInfo": Object {
              "avatar": "",
              "initials": <WithTheme(Icon)
                color="white"
                size={20}
                style={
                  Object {
                    "fontWeight": 800,
                  }
                }
              >
                check
              </WithTheme(Icon)>,
              "style": Object {
                "backgroundColor": "#80d134",
              },
            },
            "displayText": "Validated Codes",
            "firstName": "Validated",
            "lastName": "Codes",
            "userId": null,
          }
        }
        key="numeric-null"
        showAvatar={true}
      />
    </WithStyles(List)>
  </WithStyles(Collapse)>
</Fragment>
`;
